.github/workflows/publish.yml

```diff
@@ -58,7 +58,6 @@ jobs:
         if: ${{ github.event.inputs.dryrun == 'false' }}
         env:
           NPM_TOKEN: ${{ secrets.NPM_TOKEN }} # must be of type Automation to create releases
-          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
So `secrets.PAT` is used for the Checkout step in this file, but for the Publish step, there won't be any GitHub token set.

Is that because we have the `permissions.contents = write` at the beginning? In that case, is `secrets.PAT` really necessary?

Reply:
Also, `GH_TOKEN` is a Lerna specific env variable, IIRC. So if we don't have it defined anymore, it's not going to work. Right?

https://github.com/lerna/lerna/tree/main/libs/commands/version#--create-release-type

Reply:
> The PAT used for checkout persists through the whole job. It is the bot's PAT so it has the push rights needed on this step.

The CI job's PAT is actually available under `secrets.GITHUB_TOKEN`, no? I don't think it's available under `PAT`?

Reply:
Yes, but the commit and push must be authored by the bot. They can't rely on the CI job's PAT. So we basically need the bot's PAT for the commit and push and the GITHUB_TOKEN (provided by the job itself) to be used by Lerna. That is my understanding